core/space/domain/domain.go

```diff
@@ -154,5 +155,5 @@ type SharedDirEntry struct {
 	DbID   string
 	Bucket string
 	FileInfo
-	Members []Member
+	Members []Member // XXX: it is duplicated from FileInfo
```

Comment:
Okay maybe we can remove this then since it's included in FileInfo :+1:

Reply:
No that's fine you can go ahead and remove it. But we should update sharing.go `GetReceivedFiles` to move members into the file info.  I searched across the repo and it seems like that is the only reference to it :+1: